git/operations_test.go

```diff
@@ -1,11 +1,14 @@
 package git
 
 import (
-	"github.com/weaveworks/flux/cluster/kubernetes/testfiles"
+	"fmt"
```

Comment:
Thanks for reordering these, someone somewhere is not using gofmt!

Reply:
any time :)